content/blog/2023-05-10-comparison-of-ethereum-classic-and-bitcoin-monetary-policies/index.md

```diff
@@ -0,0 +1,119 @@
+
+When Ethereum (ETH) forked away from ETC in 2016, the Ethereum Classic core developer team decided to establish a monetary policy as similar as possible as that of Bitcoin. 
+
+In the table below we compare both blockchains’ monetary policies and in the following sections we will explain each feature in the table.
+
+![ETC and BTC monetary policies compared.](./etc-btc-table.png)
+
+## Digital Gold
+
+Both ETC and BTC are digital gold.
+
+The reason for this is that both networks are proof of work blockchains and the purpose of that consensus mechanism is to make it as costly to create ETC and BTC as it is in the real world for gold.
+
+The way they work is that miners on both blockchains create blocks of transactions and to stamp them cryptographically they need to do a lot of computational work, which uses a lot of electricity. Then, when the blocks are accepted by the rest of the network, the miners are credited newly issued coins by the algorithm.
+
+The above system guarantees that both ETC and BTC will be sound money as gold was in the old days, hence the term “digital gold”.
+
+## Algorithmic
+
+Bitcoin’s monetary policy was “set in stone” since it was launched in 2009. This was done by establishing its whole future distribution in the software in the form of a supply algorithm. 
+
+The supply algorithm is a set of mathematical rules that establish what number of coins will be paid to miners per block, the frequency of the payments, and a schedule that will decrease these payments as time passes by.
+
+As said before, ETC started with no monetary policy other than a fixed payment per block. However, in 2017 it migrated to a schedule that is also “set in stone” as it establishes the whole future distribution in the software in the form of a supply algorithm very similar to that of Bitcoin.
+
+## Capped Supply
+
+Both ETC and BTC have a capped supply of their coins. This means that it’s known what is the total amount that will ever exist for each blockchain.
+
+Given the frequency of payments to miners, the number of coins issued per block, and its discount schedule, Bitcoin will have a maximum supply of 21,000,000 BTC.
+
+Given Ethereum Classic’s frequency of payments to miners, the number of coins it issues per block, and its discount schedule, it will have a maximum supply of ETC of 210,700,000.
+
+Note that Ethereum Classic’s maximum supply is similar to Bitcoin’s but 10 times higher. This was done purposefully to associate both cryptocurrencies reputationally and to make it easy for the market to understand ETC’s schedule.
+
+## Block Time
+
+Ethereum Classic’s block time is 13 seconds per block and Bitcoin’s block time is 10 minutes per block.
+
+This means that ETC miners create 6,646 blocks per day and BTC miners create 144 blocks in the same period.
+
+The block time of a blockchain is important because it establishes how fast the coins will be created, thus regulates the supply. 
```

Comment:
Line 61: Correct verb form
```suggestion
The block time of a blockchain is important because it establishes how fast the coins will be created, thus regulating the supply. 
```